### ISSUE TITLE
Error building kqsl: Non-resolvable parent POM for io.confluent.ksql:ksql-parent:4.0.0-SNAPSHOT

### ISSUE THREAD

**Issue by mario-grgic** [confluentinc/ksql#481]
I'm trying to build KSQL on macOS 10.13.1, JDK 1.8.0_144 and maven 3.5.0:

```
$ mvn --version
Apache Maven 3.5.0 (ff8f5e7444045639af65f6095c62210b5713f426; 2017-04-03T15:39:06-04:00)
Maven home: /usr/local/maven
Java version: 1.8.0_144, vendor: Oracle Corporation
Java home: /Library/Java/JavaVirtualMachines/jdk1.8.0_144.jdk/Contents/Home/jre
Default locale: en_US, platform encoding: UTF-8
OS name: "mac os x", version: "10.13.1", arch: "x86_64", family: "mac"
```

The build fails with:

```
$ mvn clean compile install -DskipTests
[INFO] Scanning for projects...
[ERROR] [ERROR] Some problems were encountered while processing the POMs:
[FATAL] Non-resolvable parent POM for io.confluent.ksql:ksql-parent:4.0.0-SNAPSHOT: Failure to find io.confluent:common:pom:4.0.0-SNAPSHOT in http://packages.confluent.io/maven/ was cached in the local repository, resolution will not be reattempted until the update interval of confluent has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 7, column 13
 @
[ERROR] The build could not read 1 project -> [Help 1]
[ERROR]
[ERROR]   The project io.confluent.ksql:ksql-parent:4.0.0-SNAPSHOT (/Volumes/ramdisk/ksql/pom.xml) has 1 error
[ERROR]     Non-resolvable parent POM for io.confluent.ksql:ksql-parent:4.0.0-SNAPSHOT: Failure to find io.confluent:common:pom:4.0.0-SNAPSHOT in http://packages.confluent.io/maven/ was cached in the local repository, resolution will not be reattempted until the update interval of confluent has elapsed or updates are forced and 'parent.relativePath' points at wrong local POM @ line 7, column 13 -> [Help 2]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/ProjectBuildingException
[ERROR] [Help 2] http://cwiki.apache.org/confluence/display/MAVEN/UnresolvableModelException
```

**Comment by apurvam** [comment]
Hi @mario-grgic, sorry that you have hit this issue. We are aware that the public build is broken, and are working to fix it. In particular, we are working on changing our internal build pipelines to make the necessary -SNAPSHOT dependencies public. It should be fixed soon.

**Comment by apurvam** [comment]
This should be fixed by #490. I tested and both `master` and the `4.0.x` branch build just fine on any machine. Please reopen this issue if you continue to see the problem.

**Comment by mario-grgic** [comment]
I can also confirm this is fixed. Thank you.
